domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/showcondition/ShowConditionParser.java

```diff
@@ -15,13 +15,26 @@
 
     private static final String AND_CONDITION_REGEX = "\\sAND\\s(?=(([^\"]*\"){2})*[^\"]*$)";
     private static final String AND_OPERATOR = " AND ";
+    private static final String OR_CONDITION_REGEX = "\\sOR\\s(?=(([^\"]*\"){2})*[^\"]*$)";
+    private Pattern orConditionPattern = Pattern.compile(OR_CONDITION_REGEX);
```

Comment:
can we move this statement below the list of private static final statements?

Reply:
DONE